src/Target/TargetInterface.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Yiisoft\Yii\Debug\Target;
+
+use Yiisoft\Yii\Debug\Collector\CollectorInterface;
+
+interface TargetInterface
+{
+    public function persist(CollectorInterface $collector): void;
+
+    public function getData(): array;
```

Comment:
Do we need it to be exposed?

Reply:
Yes for first stage of developing. Later it may be removed.